packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -176,6 +293,21 @@ _Events_
     2. This event will not fire when there are no changes made to the content of the editor. For example, all mouse events, selecting the texts, state
        changes, etc,
 
+-   `mention-update`: This can be achieved through Tiptap's `onUpdate()` and `onStart()` methods in `render` function in
+    [suggestion](https://tiptap.dev/api/utilities/suggestion#render) configurations. This event fires with the `eventData` containing the
+    current text that is added after the `@` character.
+
+    This event will be triggered in the following scenarios:
+
+    1. When a user inserts the character (e.g., `@`) into the editor, which activates the mention popup.
+    2. When a user adds or removes text after inserting the mention character into the editor.
+    3. When a user repositions the cursor between the text segments added after the mention character.
+
+    Conversely, this event will not be fired when:
+
+    1. A user moves the cursor away from the mention node while the popup is open.
```

Comment:
Do we need to mention specific exit criterias here after checking in widely accepted platforms like Azdo and Github, etc.?
For ex: What does moving away a cursor mean? - In Azdo, if I enter double spaces after a word, it exits mention node whereas hitting Tab/Enter chooses the first available option for the search from dropdown

Reply:
I have updated the spec with the suggested details here